lib/msal-browser/src/index.ts

```diff
@@ -18,6 +18,7 @@ export {
 } from "./app/PublicClientApplication.js";
 export { PublicClientNext } from "./app/PublicClientNext.js";
 export { IController } from "./controllers/IController.js";
+export { StandardController } from "./controllers/StandardController.js";
```

Comment:
Please don't add these exports, these are internal to msal-browser and are not API stable, thus not intended to be public.  Let's figure out how we can enable your scenarios in a way that uses the existing public surface where possible and expose new APIs that can be properly supported where the existing APIs have gaps. Feel free to reach out if you have questions on this.